snippets/fuzzySearch.md

```diff
@@ -0,0 +1,19 @@
+### fuzzySearch
+
+Determines if the `patrn` matches with `str`
```

Comment:
Needs to be updated

`Determines if the provided pattern is matched in order within a string`